pkg/util/kubernetes/hpa/datadogexternal.go

```diff
@@ -24,14 +24,28 @@ var (
 	ddRequests = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Name: "datadog_requests",
-			Help: "Counter of requests made to datadog",
+			Help: "Counter of requests made to Datadog",
 		},
 		[]string{"status"},
 	)
+	metricsEval = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "external_metrics_processed_value",
+		Help: "value processed from querying Datadog",
+	},
+		[]string{"metric"},
+	)
+	metricsPrecision = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "external_metrics_delay",
```

Comment:
IIRC the good practice in prometheus is to suffix metric names by the unit, should we use `external_metrics_delay_seconds` instead?